config/prov/prov_test.yaml

```diff
@@ -142,30 +142,28 @@ k8s_cortx_deploy:
   git_k8_repo: "https://github.com/Seagate/cortx-k8s.git"
   git_k8_repo_file: "https://raw.githubusercontent.com/Seagate/cortx-k8s/{}/k8_cortx_cloud/{}"
   k8s_dir: "/root/deploy-scripts/k8_cortx_cloud/"
-  new_file_path: "solution.yaml"
+  new_file_path: "solution.example.yaml"
+  solution_file: "solution.yaml"
   deploy_file: "deploy-cortx-cloud.sh"
   report_file: "deployment_report.csv"
   log_file: "deployment.log"
   rgw_rpm: "cortx-rgw"
   status_log_file: "status.log"
   pre_req_log: "pre_req_log.log"
+  pre_check_log: "pre_check.log"
   deployment_type_data: "data-only"
   client_instance: 0
   cluster_yaml_path: "/etc/cortx/solution/cluster.yaml"
   cluster_conf_path: "/etc/cortx/cluster.conf"
   third_party_images:
-    consul: 'ghcr.io/seagate/consul:1.11.4'
+    consul: 'consul:latest'
```

Comment:
do we want to wait till services team agrees/provides solution?

Reply:
yes this is temporary fix